src/components/number/Number.js

```diff
@@ -178,7 +178,6 @@ export default class NumberComponent extends Input {
     const input = this.refs.input[0];
     if (input) {
       super.focus.call(this);
-      input.setSelectionRange(0, input.value.length);
```

Comment:
We would prefer that you just add a try {} catch statement around this so it is still here but does not crash IE11.

Reply:
I can do that, but just to be clear, the issue isn't IE11 crashing. Older browsers seemed to run the method without issue, it's the newer browsers that are throwing a DOMException.

Reply:
See update in new commit.